doc/man5/x509v3_config.pod

```diff
@@ -160,6 +160,10 @@ The following text names, and their intended meaning, are known:
  msCTLSign              Microsoft Trust List Signing
  msEFS                  Microsoft Encrypted File System
 
+Note that (as of 2021) there is no consensus even at standardization level
```

Comment:
See comment above.

Reply:
For me it looks like the comment below was meant;
anyway solved it in the same way.